shared/javascripts/options.js

```diff
@@ -364,7 +377,8 @@ if (Privly === undefined) {
     if (typeof glyph !== 'object' || glyph === null) {
       throw new Error('invalid argument');
     }
-    if (typeof glyph.color !== 'string' || typeof glyph.cells !== 'object' || glyph.cells === null || glyph.cells.constructor !== Array) {
+    if (typeof glyph.color !== 'string' || typeof glyph.cells !== 'object' || 
+        glyph.cells === null || Object.prototype.toString.call(glyph.cells) !== "[object Array]") {
```

Comment:
Done. I've rebased the commits so the changes can't be seen as a separate commit.